Tutorial.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Tutorial</title>
+</head>
+<body>
+	<h1>CODAP Purple air plugin Web page Tutorial</h1>
+	
+	<h2>Tutorial:</h2>
+	<p style="font-size: 18px;">This website provides information on codappurpleairdata for USA cities. To search for a specific city, enter its name in the search bar on the left. The search results will provide the city's map coordinates, and you can select the radius from that point in miles. After selecting the radius, choose the start and finish dates and press enter to access the Purple Air data. The data you will be able to access includes.</p>
+	<p style="font-size: 18px;">This is how to search any city of your wish in US:</p>
+	<img src="SSGIF.gif" alt="GIF">
+	<p style="font-size: 18px;">You will get the following Purple Air data:</p>
```

Comment:
Personally, I think we should give more details of Purple Air Data. We could say
```
<ul style="font-size: 18px;">You will get the following Purple Air data:
    <li>Map: visualize the location of selected sensors in the city's map</li>
    <li>Case Table: visualize the data in the date range and compare to other cities</li>
</ul>
```
Above is my idea